grails-app/controllers/org/pih/warehouse/inventory/InventoryController.groovy

```diff
@@ -565,6 +565,66 @@ class InventoryController {
         render(view: "list", model: [availableItems: inventoryItems])
     }
 
+    def reorderReport = {
+        Location location = Location.get(session.warehouse.id)
+        def inventoryItems = dashboardService.getReorderReport(location)
+
+        String filename = "Reorder report - " + location.name + ".csv"
+
+        response.setHeader("Content-disposition", "attachment; filename=\"${filename}\"")
+
+        def hasRoleFinance = userService.hasRoleFinance(session.user)
+
+        def csv = ""
+        csv += '"' + "${warehouse.message(code: 'inventoryLevel.status.label')}" + '"' + ","
```

Comment:
Probably would have been better to use the dataService.generateCsv() and use StringEscapeUtils.escapeCsv() for wrapping all values in double quotes. But I assume we were working off an older version that I wrote so I'll let it pass.